.github/workflows/create-installer-iso.yml

```
@@ -17,6 +17,12 @@ jobs:
       - name: Install Nix
         uses: cachix/install-nix-action@v31
 
+      - name: Setup cachix
+        uses: cachix/cachix-action@v16
+        with:
+          name: holochain-ci
+          authToken: "${{ inputs.CACHIX_AUTH_TOKEN }}"
```

actionlint (1.7.8)

property "cachix_auth_token" is not defined in object type {}

(expression)